Scala/Scala.sublime-syntax

```diff
@@ -813,6 +857,7 @@ contexts:
       scope: support.type.scala
     - include: base-type-expression
   single-type-expression:
+    - include: annotation
```

Comment:
This isn't going to work quite correctly, and also I think it's probably wrong.  For example:

```scala
x: @specialized Foo
```

Does that parse?  I actually don't know, but I don't think it does.  

Anyway, the reason it won't work correctly is because the space following `@specialized` will break out of the `single-type-expression` context!  If the above *does* parse, then you need to have a bit more machinery that parses the annotation, consumes the whitespace and then `set`s back into `single-type-expression`.  It isn't valid to ascribe *just* an annotation as a type, so that probably works.